src/Resources/contao/languages/de/modules.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+
+/**
+ * Extension folder
+ */
+$GLOBALS['TL_LANG']['MOD']['autoregistration'] = array('Nach Registrierung automatisch aktivieren/anmelden');
```

Comment:
This is no longer required, it was only used in the backend settings for disabled modules, but that does no longer exist in Contao 4